### PR TITLE
Pass file containing client hostnames directly to fio

### PR DESCRIPTION
**Pull request by ndokos**
This addresses issues #358 and (incidentally and opportunistically) #341.

### CODE REVIEW

**Comment by ndokos** [comment]
@portante: can you please review?

File: agent/bench-scripts/pbench-fio [hunk 1]
```diff
@@ -501,7 +512,11 @@ function fio_run_job() {
 	fi
 	pbench-start-tools --group=$tool_group --iteration=$iteration --dir=$benchmark_results_dir
 	local client_opts=""
-	if [ ! -z "$clients" ]; then
+
+        if [ ! -z "$client_file" ] ;then
```
Comment (portante) [inline agent/bench-scripts/pbench-fio]: Small indentation problem.

**Comment by portante** [comment]
Looks good otherwise.

File: agent/bench-scripts/pbench-fio (hunk 1 above)
Comment (ndokos) [inline agent/bench-scripts/pbench-fio]: Fixed.

**Pull request merged**